client/settings/general-settings-section/__tests__/general-settings-section.test.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import GeneralSettingsSection from '..';
+
+describe( 'GeneralSettingsSection', () => {
+	it( 'should render the card information', () => {
+		render( <GeneralSettingsSection /> );
+
+		expect(
+			screen.queryByText( 'Credit card / debit card' )
+		).toBeInTheDocument();
+		expect(
+			screen.queryByText(
+				'Let your customers pay with major credit and debit cards without leaving your store.'
+			)
+		).toBeInTheDocument();
+	} );
+
+	it( 'should render the opt-in banner with action elements', () => {
+		render( <GeneralSettingsSection /> );
+
+		expect( screen.queryByTestId( 'opt-in-banner' ) ).toBeInTheDocument();
+		expect(
+			screen.queryByText( 'Enable in your store' )
```

Comment:
This is 100% OK but for a little extra benefit you could also check if what is rendered is a button:

```suggestion
			screen.queryByRole( 'button', { name: 'Enable in your store' } )
```

Reply:
This element is found by role `link`. Updated the test with this-
`screen.queryByRole( 'link', { name: 'Enable in your store' } )`